src/GovUk.Education.ExploreEducationStatistics.Content.Services/Interfaces/IDataSetFileService.cs

```diff
@@ -29,5 +29,5 @@ public interface IDataSetFileService
     Task<Either<ActionResult, DataSetFileViewModel>> GetDataSetFile(
         Guid dataSetId);
 
-    Task<Either<ActionResult, List<DataSetSitemapItemViewModel>>> ListSitemapItems();
+    Task<Either<ActionResult, List<DataSetSitemapItemViewModel>>> ListSitemapItems(CancellationToken cancellationToken);
```

Comment:
To simplify the tests, we've typically been adding `default` for the cancellation tokens i.e.

```cs
ListSitemapItems(CancellationToken canellationToken = default);
```

This way, we don't need to call the service methods with this argument as it's not relevant most of the time.

This can be done with the other interfaces and the implementations as well, and then the tests can be simplified too